lerna.json

```
@@ -0,0 +1,15 @@
+{
+  "packages": [
+    "packages/**",
+    "samples/*"
+  ],
+  "version": "independent",
+  "npmClient": "yarn",
+  "command": {
+    "version": {
+      "ignoreChanges": ["**/__tests__/**", "**/*.unit.spec.ts"],
```

normally this shouldn't happen because we use `--conventional-commits`,
but if you really want to use it then we could add md files at least
```suggestion
      "ignoreChanges": ["**/*.md", "**/*.test.ts", "**/*.spec.ts"],
```

---

If you change the readme and want that to be reflected on npm, then we should maybe just let people use conventional commits to do that if they like. I agree that bumping versions on npm because of a readme is weird, but bumping due to tests changing is probably worse, as the tests are in no way included in the package.

---

we definitely won't bump the patch version just because of readme (I've never heard ppl doing that), it'll wait until the next release
so for me it's no difference between readme and tests tbh

---

so it's just up to developers if they're following the semantic versioning (which they must)

---

I added *.md

Unrelated: Should we not use `**/__test__/**` as a check for tests?
I know jest only cares about `.test.ts(x?)` and `.spec.ts(x?)`, but there can be other files related to tests in __test__, so I think we should keep it ignored, both in lerna and in tsconfig.

On that note: Do we ever use `.tsx` tests? They would be picked up by jest but not lerna, so we should probably make them agree

---

doesn't matter really, can be changed later